src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -388,9 +388,9 @@ LibraryServiceClient.prototype.createShelf = function(request, options, callback
  *
  * @param {Object} request
  *   The request object that will be sent.
- * @param {string} request.name
+ * @param {String} request.name
```

Comment:
Hmm, this is tiny and maybe harmless update, but we actually prefer lower-cased `string` rather than `String`. It used to be `String` but I changed to `string`.  Probably your code started referring some piece of code unchanged since then.

Please update the name reference to `string` with small s.